src/applications/check-in/utils/appConstants/appConstants.unit.spec.js

```diff
@@ -48,4 +48,35 @@ describe('check in utils', () => {
       ).to.equal(expected);
     });
   });
+
+  describe('isInAllowList', () => {
+    it('should return false if station not in list', () => {
+      const appointment = {
+        stationNo: '0002',
+        clinicIen: '0001',
+      };
+      expect(isInAllowList(appointment)).to.be.false;
+    });
+    it('should return false if clinic not in list', () => {
+      const appointment = {
+        stationNo: '0001',
+        clinicIen: '0002',
+      };
+      expect(isInAllowList(appointment)).to.be.false;
+    });
+    it('should return true if clinic, and station match', () => {
+      const appointment = {
+        stationNo: '0001',
+        clinicIen: '0001',
+      };
+      expect(isInAllowList(appointment)).to.be.true;
+    });
+    it('should return true only station listed', () => {
+      const appointment = {
+        stationNo: '500',
+        clinicIen: null,
```

Comment:
Shouldn't `clinicIen` have a non-null value here? It should always have one in LoROTA so I think that's how we would want to validate it.

Reply:
Yeah that makes sense. I think my line of thought was something where it clearly wasn't being evaluated. Update pushed!